language/move-ir-compiler/transactional-tests/tests/parsing/structs.mvir

```diff
@@ -0,0 +1,13 @@
+//# print-bytecode --input=module
+// A struct declaration cannot contain function declarations (like the syntax for declaring
+// "methods" in other languages).
+module 0x3d8.M {
```

Comment:
It seems like you've been using different addresses at different places. Just curious: is there any specific reason you're doing this? They seem to follow some sort of pattern...

Reply:
No meaningful pattern in terms of program semantics, but they do follow a pattern: dice combinations for Dungeons and Dragons-style tabletop games: this one is "three 8-sided dice," others are "two 20-sided dice" 🤓